src/stories/LossSelection.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ComponentMeta } from '@storybook/react';
+import HistorySelection, { HistorySelectionProps } from './HsitorySelect';
+
+export default {
+    component: HistorySelection,
+    title: 'History Selection/HistorySelection',

```

Comment:
HistorySelection/History Selection - How about to rename it to `Components/History Selection` to move it in the right stories category?